internal/bminventory/inventory.go

```diff
@@ -3131,6 +3135,11 @@ func logReplyReceived(params installer.PostStepReplyParams, log logrus.FieldLogg
 	}
 }
 
+func shouldBeLogged(params installer.PostStepReplyParams, host *common.Host) bool{
+	return !(params.Reply.ExitCode == MediaDisconnected && *host.Status == string(models.HostStageFailed) &&
+		strings.Contains(*host.StatusInfo, mediaDisconnectionMessage))
```

Comment:
Why do we need to check both the host.StatusInfo and the MediaDisconnected exit code?
If we got media disconnection, and we are in failed stage, than that is enough, no?

Reply:
If the installation has failed the assisted-installer itself reports an error (not indicative) and change the status  and afterward the agent reports the error (with the media disconnection indicative error) so i want it to be logged but only one time.